src/scaffoldmaker/meshtypes/meshtype_3d_bladder1.py

```diff
@@ -614,14 +608,14 @@ def generateBaseMesh(cls, region, options):
                                                               get_bladder_term("right ureter junction with bladder"))
 
         idx1 = 1
-        xi1 = [0.0, 0.0, 1.0]
+        xi1 = [0.0, 0.0, 0.0]
         markerList = []
         markerList.append({"group": apexGroup, "elementId": idx1, "xi": xi1})
         idx2 = elementsCountAlongDome * elementsCountAround * elementsCountThroughWall + ureterElementPositionAround + 1
-        xi2 = [ureter1Position.xi1, 0.0, 1.0]
+        xi2 = [ureter1Position.xi1, 0.0, 0.0]
```

Comment:
This should have remained at 1.0 ? Want apex marker on the outside. Check it is correct and the bladder test still passes!

Reply:
I moved the apex to outside surface and updated the test. Done.